app/registries/foreman/plugin.rb

```diff
@@ -191,6 +191,7 @@ def initialize(id)
       @ping_extension = nil
       @status_extension = nil
       @allowed_registration_vars = []
+      @observable_events = []
```

Comment:
How will we access this? Will `EventSubscribers` read this?

Reply:
Yeah I noticed too late in the evening :-)